assets/staffportal/json/application/information-form/application-information-form-json-nefba2.json

```diff
@@ -0,0 +1,127 @@
+                            "name": "quartech_lengthoftimeinbusinessinmonths",
+                            "visible": false
+                        }
+                    ]
+                },
+                {
+                    "name": "vpswppAnimalSpeciesTypeServedSection",
+                    "visible": false
+                }
+            ]
+        },
+        {
+            "name": "applicationReviewTab",
+            "sections": [
+                {
+                    "name": "documentSubmissionSection",
+                    "visible": false
+                },
+                {
+                    "name": "eligibilitySection",
+                    "fields": [
+                        {
+                            "name": "quartech_vetpracticeaccreditedbycvbc",
+                            "visible": false
+                        }
+                    ]
+                },
+                {
+                    "name": "evaluationSection",
+                    "fields": [
+                        {
+                            "name": "quartech_review1outcome",
+                            "visible": true
+                        },
+                        {
+                            "name": "quartech_numberofstudentsapproved",
+                            "visible": false
+                        },
+                        {
+                            "name": "quartech_reviewer1score",
+                            "visible": false
+                        },
+                        {
+                            "name": "quartech_tr2_completedon",
+                            "visible": false
+                        },
+                        {
+                            "name": "quartech_reviewer2score",
+                            "visible": false
+                        },
+                        {
+                            "name": "quartech_tr3_completedon",
+                            "visible": false
+                        },
+                        {
+                            "name": "quartech_reviewer3score",
+                            "visible": false
+                        },
+                        {
+                            "name": "quartech_finalscore",
+                            "visible": false
+                        },
+                        {
+                            "name": "quartech_approvaldiscussioncompletedon",
+                            "visible": false
+                        },
+                        {
+                            "name": "quartech_reviewcommitteediscussioncompletedon",
+                            "visible": false
+                        },
+                        {
+                            "name": "quartech_followupquestions",
+                            "visible": false
+                        }
+                    ]
+                },
+                {
+                    "name": "vpswppStudent1Section",
```

Comment:
👍normally these sections are hidden by default but it 's good to explicitly add it here